docs/src/release-notes-js.md

```diff
@@ -6,6 +6,81 @@ toc_max_heading_level: 2
 
 import LiteYouTube from '@site/src/components/LiteYouTube';
 
+## Version 1.46
+
+### TLS Client Certificates
+
+TLS Client Authentication allows the server to request a client certificate and verify it.
+This release allows you to specify them for the browser and when using the [APIRequestContext] (API Testing).
```

Comment:
```suggestion
TLS Client Authentication is a web security measure where clients authenticate themselves to the server using a client certificate.
This release adds support for client certificates in Playwright tests.
```

The fact that the server "requests the client certificate" is an implementation detail that the changelog audience doesn't care about. I've added the fact that this is about web security so it's easier for folks to understand, who have never heard about "TLS" before.

Also, you're talking about `APIRequestContext` here, but also about a browser context below? I'm assuming this works for both, so i've removed the mention.

Reply:
web-security sounds scary, I'd leave it out for now.

There were two dedicated feature requests, hence mentioning both separately.